packages/gateway/src/durable-objects/gateway-metrics.js

```diff
@@ -4,20 +4,24 @@
  * @property {number} totalSuccessfulRequests total number of successful requests
  * @property {number} totalFailedRequests total number of requests failed
  * @property {number} totalWinnerRequests number of performed requests where winner
+ * @property {number} totalRateLimitedErroredRequests number of performed requests prevented from rate limiting
+ * @property {number} totalRateLimitedPreventedRequests number of performed requests prevented from rate limiting
  * @property {Record<string, number>} responseTimeHistogram
  *
  * @typedef {Object} ResponseStats
  * @property {boolean} ok request was successful
  * @property {number} [responseTime] number of milliseconds to get response
- * @property {boolean} [winner]
+ * @property {boolean} [winner] response was from winner gateway
+ * @property {boolean} [rateLimitPrevented] response not obtained as request did not happen to prevent rate limit blockage
```

Comment:
```suggestion
 * @property {boolean} [rateLimited] request not sent to upstream gateway as it exceeds the rate limit
```

Reply:
Changing to

```js
@property {boolean} [rateLimitPrevented] request not sent to upstream gateway as it would exceed the rate limit
```

I think `rateLimited` is not the wording because we were not rate limited by the gateway, but we prevented a request from triggering a rate limit to us. This is why I called `prevented` as we kept it from happening. What do you think?